adoc/extensions/sycl_khr_includes.adoc

```diff
@@ -0,0 +1,610 @@
+
+The Description column in the table is non-formative and intended to help you
+navigate the extension document quicker. Formal description of each new header
+file is provided in subsequent sections below.
+
+[[table.khr-includes-list]]
+.List of headers provided by the extension
+[width="100%",options="header",cols="1,2"]
+|====
+|Header
+|Description
+
+|[code]#<sycl/khr/includes/version>#
+|This header contains all macro definitions provided by the implementation
+
+|[code]#<sycl/khr/includes/submit>#
+|This header covers queue operations. It contains definitions of [code]#queue#,
+[code]#handler# and [code]#event# class. If you don't use events and/or only
+use [code]#queue# shortcut functions, then it is recommended to include those
+classes individually through the corresponding headers.
+
+|[code]#<sycl/khr/includes/queue>#
+|Contains definition of [code]#queue# class.
+
+|[code]#<sycl/khr/includes/handler>#
+|Contains definition of [code]#handler# class.
+
+|[code]#<sycl/khr/includes/event>#
+|Contains definition of [code]#event# class.
+
+|[code]#<sycl/khr/includes/runtime>#
+|Contains definitions of [code]#device#, [code]#platform# and [code]#context#
+classes, as well as device selectors.
+
+|[code]#<sycl/khr/includes/index_space>#
+|Contains definitions of [code]#id#, [code]#range#, [code]#nd_item#,
+[code]#nd_range# and [code]#item# classes
+
+|[code]#<sycl/khr/includes/usm>#
+|Contains definition of [code]#usm_allocator# class and USM free functions for
+memory management (like [code]#malloc_shared#).
+
+|[code]#<sycl/khr/includes/buffer>#
+|Contains definition of [code]#buffer# class
+
+|[code]#<sycl/khr/includes/accessor>#
+|Contains definitions of [code]#accessor#, [code]#local_accessor# and
+[code]#host_accessor# classes.
+
+|[code]#<sycl/khr/includes/atomic>#
+|Contains definition of [code]#atomic_ref# class
+
+|[code]#<sycl/khr/includes/half>#
+|Contains definition of [code]#half# class
+
+|[code]#<sycl/khr/includes/vec>#
+|Contains definition of [code]#vec# class
+
+|[code]#<sycl/khr/includes/marray>#
+|Contains definition of [code]#marray# class
+
+|[code]#<sycl/khr/includes/math>#
+|This header provides access to the corresponding group of built-in
+functions defined by <<sycl:builtins>>
+
+|[code]#<sycl/khr/includes/groups>#
+|This header provides access to [code]#group# and [code]#sub_group# classes, as
+well as operations that can be performed with them such as broadcasts or scans
+
+|[code]#<sycl/khr/includes/multi_ptr>#
+|This header provides access to [code]#multi_ptr# class and related address
+space class aliases
+
+|[code]#<sycl/khr/includes/functional>#
+|This header provides access to function objects like [code]#plus# or
+[code]#locial_or# defined by <<sec:function-objects>>
+
+|[code]#<sycl/khr/includes/reduction>#
+|This header provides access to functionality related to reductions (such as
+[code]#reduction# interface and [code]#reducer class)
```

Comment:
```suggestion
[code]#reduction# interface and [code]#reducer# class)
```

Reply:
Applied in 2f70d45aaef845bd58f590cd75eda3dea70dc932